### PR TITLE
Fixes #2

### LINKED ISSUE #2: Yarn Is not Correctly Detected on Windows
Windows does not support using spawn for anything other than `.exe` files, unless the option {shell:true} is used. However, that is not supported for any node version `< 6.0.0`. [cross-spawn](https://www.npmjs.com/package/cross-spawn) is a drop-in replacement that is cross compatible across all node versions and platforms and would allow for this package to work properly.

Due to the above failure, yarn is not correctly detected. If the code also tested in the same way for npm, it will fail the same, it is **not** an issue with yarn, but the underlying child_process API.

### PR DESCRIPTION
**Pull request by memchk**
This fixes issue #2 by using cross-spawn. Verified to work with both yarn and npm.

### CODE REVIEW

**Review by shvaikalesh — CHANGES_REQUESTED** [review]
`cross-spawn` is too heavy. Dropping Node's < 6 is ok.

**Comment by memchk** [comment]
Done. Since shell = false was known to work for UNIX platforms I also added a conditional to check if the platform was windows. I would recommend bumping the version and also updating brunch/brunch as that is how I found this problem.

File: index.js [hunk 1]
```diff
@@ -13,7 +12,9 @@ process.env.FORCE_COLOR = 'true';
 process.env.NPM_CONFIG_COLOR = 'always';
 
 const installed = cmd => {
-  return !spawn.sync(cmd, ['--version']).error;
+  // shell: true must be set for this test to work on non *nixes.
+  const useShell = process.platform === "win32" ? true : false;
```
Comment (shvaikalesh) [inline index.js]: Let's drop unneeded ternary here:
```js
const shell = process.platform === 'win32'
return !cp.spawnSync(cmd, ['--version'], {shell}).error;
```

**Comment by shvaikalesh** [comment]
And please let's not include `yarn.lock`

**Comment by memchk** [comment]
Alrighty. I already removed yarn.lock and added it the .gitignore since devs may leave it in there accidentally while testing. (What I did)

**Comment by shvaikalesh** [comment]
Thanks for the PR and patience while making changes, @r3liant.

@paulmillr could you please release patch?

**Pull request merged**